src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/HashHelper.java

```diff
@@ -17,6 +17,10 @@ public String encode(String plain) throws NoSuchAlgorithmException {
         return stripEquals(base64Encode(sha1Digest(plain)));
     }
 
+    public String generateHashedId(String id) throws NoSuchAlgorithmException {
```

Comment:
This is just a rehash of encode. Can we not just use encode()?

Reply:
No need for this extra method

Reply:
Extra method removed, now using encode